src/Visualization/Python/GenerateXdmf.py

```diff
@@ -246,6 +246,10 @@ def data_item(data_type):
                 components.remove('grid_names')
```

Comment:
Would something like a `list` of things to remove if they exist be useful/more maintainable?

Reply:
🤷‍♂️ I think we should eventually use the Py bindings here. Then we don't need to maintain this list in two places at all (`h5::VolumeData::list_tensor_components` already handles this).

Reply:
Ah yea, agreed :)